mrp_repair_reorder_lines/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Repair Lines with sequence number
+=================================
+
+This module was written to allow to manage the repair lines order.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
Put this inside https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#configuration section